app/generators/contestantRoundListGenerator.tsx

```diff
@@ -14,35 +13,35 @@ interface Dictionary<T> {
 export default async function generateListOfContestantRoundLists(
     dataFetcher: () => Promise<ITableRowData[]>,
     listOfContestantLeagueData: any[],
-    getCompetingEntityListFunction: (x: ITableRowData[]) => any = getTeamList,
+    getCompetingEntityListFunction: (_x: ITableRowData[]) => any = getTeamList,
```

Comment:
As discussed in person `_` is probably preferable because this isn't even a variable that is "accessible". Of course in order to permit this we will need to update the regex one more time.

Reply:
Done!